cabal.project

```diff
@@ -37,3 +38,9 @@ packages:
     deps/argo/argo
     deps/argo/tasty-script-exitcode
     deps/cryptol/cryptol-remote-api
+    deps/language-rust
+
+source-repository-package
```

Comment:
I've been wondering whether we might want to switch to this mode for all of our git-located dependencies, but it's a bit weird to have a mix of this and submodules. Maybe this could be made into a submodule for now, for consistency, and then we could experiment with putting all of our git dependencies in `cabal.project` in a later PR.

Reply:
I'll ask about getting a hobbits release.

Reply:
For `language-rust`, I think it might be being used by another SAW submodule...? I don't know. I think we are just using the version on hackage, so maybe it doesn't need to be in `deps`, but I am not completely sure.

As for Hobbits, I could imagine making it a submodule. I was originally hoping that we would update it and be done, but it does seem like we have been doing some amount of development on it. More generally, I would like (in the next, maybe, 12 months) to write another Hobbits paper, so I would like to make it release-able at some point, but that is not necessarily incompatible with making it a submodule, just something to consider.